controllers/managedmcg_controller.go

```diff
@@ -177,25 +183,55 @@ func (r *ManagedMCGReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 func (r *ManagedMCGReconciler) reconcilePhases() (reconcile.Result, error) {
 	r.Log.Info("Reconciler phase started..&&")
 
+	initiateUninstall := r.checkUninstallCondition()
 	// Update the status of the components
 	r.updateComponentStatus()
 
 	if !r.managedMCG.DeletionTimestamp.IsZero() {
 		r.Log.Info("removing managedMCG resource if DeletionTimestamp exceeded")
 		if r.verifyComponentsDoNotExist() {
 			r.Log.Info("removing finalizer from the ManagedMCG resource")
-			r.managedMCG.SetFinalizers(utils.Remove(r.managedMCG.GetFinalizers(), ManagedMCGFinalizer))
+
+			// Deleting Noobaa CSV from the namespace
+			/*r.Log.Info("deleting Noobaa CSV")
+			if csv, err := r.getCSVByPrefix(noobaaOperatorName); err == nil {
+				if err := r.delete(csv); err != nil {
+					return ctrl.Result{}, fmt.Errorf("unable to delete csv: %v", err)
+				}
+			} else {
+				return ctrl.Result{}, err
+			}*/
```

Comment:
We do need to delete the CSV as well, right?

Reply:
@rexagod  we need to delete mcg CSV and we are doing it in removeOLMComponents() method

Reply:
Ah okay, in that case can we remove this comment?